robot/rover/comms/src/CommandCenter.cpp

```diff
@@ -0,0 +1,29 @@
+//
+// Created by cedric on 2020-10-14.
+//
+
+#include "include/CommandCenter.h"
+
+namespace internal_comms
+{
+    Command* CommandCenter::processCommand() const
+    {
+        uint8_t commandID = Serial.read();
+        size_t bytesToRead = Serial.read();
+        uint8_t* buffer = (uint8_t*) malloc(sizeof(uint8_t) * bytesToRead);
+        uint8_t bytesRead = (uint8_t) Serial.readBytes((char*)buffer, bytesRead);
```

Comment:
Not sure if it is necessary to have an args length, since when each message is received (depending on the commandID type) there will be a fixed number of args for that specific command type (ex. fan_on uses only 1 arg, therefore reading the message will only read from 1 arg be default, and not require argsLength to say how many args to read from). 

Unless I am misunderstanding something.

Reply:
The purpose of this PR is just to create a basic architecture for you guys to work with. I just added these lines for you to know where to read/write commands. The details of how the commands are set should be tackled in the issue after #498

Reply:
@MartensCedric Gotcha. I don't have anything else to add.